test/uk/gov/hmrc/gform/sharedmodel/FormComponentRejectSpec.scala

```diff
@@ -123,7 +125,6 @@ class FormComponentRejectSpec extends Spec with TableDrivenPropertyChecks {
       ("json",                                     "expectedMessage"),
       ("invalid-count-usage",                      "sections.fields.[id=lastInfo].infoText: textA cannot be use with .count function. Only AddToList id can be used with .count"),
       ("invalid-sum-usage",                        "sections.fields.[id=lastInfo].infoText: textA cannot be use with .sum function. Only numeric fields from Group component, Repeated section or AddToList section can be used with .sum function"),
-      ("invalid-addtolist-cross-reference",        "title: fieldA belongs to different Add To List section"),
```

Comment:
File invalid-addtolist-cross-reference.json should be deleted, as no longer needed.